apps/docs/src/.vitepress/components/OnyxTypography.stories.ts

```diff
@@ -0,0 +1,45 @@
+import { defineStorybookActionsAndVModels } from "@sit-onyx/storybook-utils";
+import type { Meta, StoryObj } from "@storybook/vue3";
+import OnyxTypography, { type TypographyToken } from "./OnyxTypography.vue";
+
+const meta: Meta<typeof OnyxTypography> = {
+  title: "components/OnyxTypography",
+  ...defineStorybookActionsAndVModels({
+    component: OnyxTypography,
+    events: [],
+  }),
+};
+
+export default meta;
+type Story = StoryObj<typeof OnyxTypography>;
+
+export const Headlines = {
+  args: {
+    tokens: Array.from<unknown, TypographyToken>({ length: 6 }, (_, index) => {
+      const name = `h${index + 1}`;
+      return { name, htmlTag: name as TypographyToken["htmlTag"] };
+    }),
+  },
+} satisfies Story;
+
+export const Paragraphs = {
+  args: {
+    tokens: [
+      { name: "paragraph-big", fontSize: "big", htmlTag: "p" },
+      { name: "paragraph-default", fontSize: "default", htmlTag: "p" },
+      { name: "paragraph-small", fontSize: "small", htmlTag: "p" },
+    ],
+    wideName: true,
+  },
+} satisfies Story;
+
+export const Links = {
```

Comment:
<img width="369" alt="image" src="https://github.com/SchwarzIT/onyx/assets/151019977/293a8ba4-7b6d-4d07-88fc-8be4478c6edc">


the link example does not look right :O

Reply:
solved with the latest state